src/main/scala/Afe.scala

```diff
@@ -0,0 +1,94 @@
+package edu.berkeley.cs.ucie.digital
+
+import chisel3._
+import chisel3.util._
+
+class StandardPackageIo(lanes: Int = 16) extends Bundle {
+  // TX
+  val txData = Output(Bits(lanes.W))
+  val txValid = Output(Bool())
+  val txTrack = Output(Bool())
+  val txClkp = Output(Bool())
+  val txClkn = Output(Bool())
+
+  // RX
+  val rxData = Input(Bits(lanes.W))
+  val rxValid = Input(Bool())
+  val rxTrack = Input(Bool())
+  val rxClkp = Input(Bool())
+  val rxClkn = Input(Bool())
+
+  // Sideband
+  val txDataSb = Output(Bool())
+  val rxDataSb = Input(Bool())
+  val txClkSb = Output(Bool())
+  val rxClkSb = Input(Bool())
+}
+
+// The analog front-end (AFE) interface, from the perspective of the logical PHY layer.
+class AfeIo(
```

Comment:
"Upward-facing" IOs (stuff that talks to digital) should be declared in a separate bundle so they can be directly reused by upper layers. There's probably no real reason to have a single bundle containing both the downwards (analog) IOs and upwards IOs, since chisel can handle multiple IOs in a module anyways.